### ISSUE TITLE
v3.9.0+ breaks on node v0.12

### ISSUE THREAD

**Issue by ljharb** [zloirock/core-js#933]
`require('core-js')` breaks in node v0.12 with:
```
TypeError: Cannot convert a Symbol value to a string
    at String (native)
    at Object.<anonymous> ($PWD/node_modules/core-js/modules/es.symbol.description.js:34:23)
```

This works fine in v3.8.3, and breaks in v3.9.0. Looking at https://github.com/zloirock/core-js/compare/v3.8.3...v3.9.0, but nothing jumps out.

**Comment by zloirock** [comment]
Interesting, thanks, I'll check it tomorrow.

**Comment by zloirock** [comment]
Are you sure that it's introduced not in 3.9.1? It could be caused by https://github.com/zloirock/core-js/commit/2394dacaa5937e77e692a0821424322ed2578e9c .

**Comment by ljharb** [comment]
I’m not certain; i installed a lot of versions to try to isolate it. That commit does seem the likely cause.

**Comment by zloirock** [comment]
Thanks, seems at the time of this PR I forgot about old V8 versions shema. I'll publish it after testing.

**Comment by zloirock** [comment]
Fixed in 3.12.0.
